spec/factories/users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :user do
+    sequence(:email) { |n| "hello+#{n}@test.com" }
+    password { 'password' }
+    password_confirmation { 'password' }
+  end
+end
```

Comment:
RSpec.describe 'recipes#index', type: :feature do
  describe 'recipe food' do
    before(:each) do

end

Reply:
Hi @ismailakinkunmi,

I did include RSpec and used capybara for end-to-end testing on the Food page, The code you're showing me is for authentication so the test can access to food page if I don't include the FactoryBot to create a user. the test will be redirected to the sign_in page.
The file for the tests is in the system folder under the name `foods_spec.rb`, kindly recheck this,

Reply:
![Screenshot_20220803_111855](https://user-images.githubusercontent.com/25227022/182585021-2264beb3-a14d-4447-a414-30c2f3045b96.png)

Reply:
@Ismailco ok noted I re-run and get back to you